genenetwork/genenetwork3
# Heatmap generation

#### Description
Provide the ability for the code to generate heatmaps in a horizontal, or vertical orientation.

#### How should this be tested?
One could run the server, and then run a cURL command against it to see the output image that is generated. As an example, see 
[test_heatmaps_rework.py.zip](https://github.com/genenetwork/genenetwork3/files/7295298/test_heatmaps_rework.py.zip) to see how one would probably go about testing this.

#### Any background context you want to provide?
https://github.com/genenetwork/gn-gemtext-threads/blob/main/topics/gn1-migration-to-gn2/non-clustered-heatmaps-and-flipping.gmi

#### What are the relevant pivotal tracker stories?
https://github.com/genenetwork/gn-gemtext-threads/blob/main/topics/gn1-migration-to-gn2/non-clustered-heatmaps-and-flipping.gmi

#### Screenshots (if appropriate)

#### Questions
N/A
